runner/run_benchmark.py

```diff
@@ -142,25 +154,58 @@ def execute_command(command):
             cmd.extend(i)
         else:
             cmd.append(str(i))
+    return cmd
+
+
+def enqueue_output(out, queue):
+    while True:
+        line = out.readline()
+        queue.put(line)
+        if not line:
+            break
+    out.close()
 
+
+def execute_command(command, timeout=None, stacktrace=False):
```

Comment:
Could you provide some context on why you changes this and what exactly it does?

Reply:
I wanted to read the output of the process in a non-blocking way so that we could capture the stack trace of the running process before killing it. By default, the stack trace is captured because the user may not have eu-stack, but I wanted this feature so that ideally, it might be possible to debug a deadlock in CI without having to manually run the executable locally hundreds of times.